tools/istio-iptables/pkg/cmd/run.go

```diff
@@ -483,7 +485,7 @@ func (iptConfigurator *IptablesConfigurator) executeIptablesRestoreCommand(isIpv
 		return err
 	}
 	// --noflush to prevent flushing/deleting previous contents from table
-	iptConfigurator.ext.RunOrFail(cmd, "--noflush", rulesFile.Name())
+	iptConfigurator.ext.Run(cmd, "--noflush", rulesFile.Name())
```

Comment:
Revert. This change is neither needed nor related to the rest of this PR.

Reply:
done

Reply:
reverted